test/src/org/labkey/test/tests/targetedms/TargetedMSChromatogramOptimizationTest.java

```diff
@@ -0,0 +1,126 @@
+package org.labkey.test.tests.targetedms;
+
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.categories.MS2;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.targetedms.ConnectionSource;
+
+import java.io.File;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Category({DailyB.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 6)
```

Comment:
Is this timeout meaningful? Can we just leave it with the default value?

Reply:
The test took 2.14 sec to run on teamcity, so changed the timeout to 3.
https://teamcity.labkey.org/viewLog.html?buildId=1245005&buildTypeId=LabkeyTrunk_ModuleSuites_TargettedMSPostgres&tab=testsInfo